fishtest/fishtest/rundb.py

```diff
@@ -116,7 +116,7 @@ def new_run(self, base_tag, new_tag, num_games, tc, book, book_depth, threads, b
 
   def get_machines(self):
     machines = []
-    for run in self.runs.find({'tasks': {'$elemMatch': {'active': True}}}):
+    for run in self.runs.find({'tasks.active': True}):
```

Comment:
If changed here we should change all `$elemMatch` for consistency.

I prefer to leave it unchanged

Reply:
My concern is that the elemmatch is a more complex operation and may be slower than the simpler test. But I don't know how it's implemented, it's possible the query optimiser sees through the un-necessary complexity and implements both operations the same way under the hood.
I will remove the rundb.py change for now while we investigate ...